lib/linguist/languages.yml

```diff
@@ -44,6 +44,16 @@ AGS Script:
   tm_scope: source.c++
   ace_mode: c_cpp
 
+Alex Haskell:
+  type: programming
+  group: Haskell
```

Comment:
If you make Akex Haskell part of the Haskell group it will be displayed in the statistics and in search results as being Haskell... Is that what you want?